go/private/actions/stdlib.bzl

```diff
@@ -82,6 +82,9 @@ def _build_stdlib(go):
     if go.mode.race:
         args.add("-race")
     args.add_all(go.sdk.experiments, before_each = "-experiment")
+    args.add("-package", "std")
+    if not go.mode.pure:
+        args.add("-package", "runtime/cgo")
```

Comment:
Does this unconditionally include `runtime/cgo` in the binary? If that isn't used do you get an unnecessary import/link directive error?

Reply:
No, this just affects the `stdlib` install action, it won't link anything extra into binaries. The term `package` is slightly confusing here, but it is what the arguments to `go install` are (with `std` being a special value).